### DISCUSSION TITLE
problem with editor on android devices

### DISCUSSION THREAD

**Discussion by lolihanter1228** [ppy/osu#23221] [Q&A] [answered]
on Android devices it is impossible to create a slider or spinner, you can start them but you can’t complete them if possible then tell me how I wait a lot of time until they fix it or tell me how

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/19422

**Comment by lolihanter1228** [comment]
sorry for bothering me and my obsession but i tried everything and long press and double tap nothing works
